FutureNorthants/northants-website
# Example solution for #75, hide alerts on specific pages

**Drupal changes**

(don't apply these straight away, see testing first)
1.  Go to Structure | Content types | Service page | Manage fields
2. Add a new boolean field, _field_hide_alert_, defaulting to not checked, label "_Hide alert message_" and not a required field.
3. Go to Structure | Content types | Service page | Manage form display
4. Drag your new field up to under the "Parent" field in the "Main" tab and save.

**Testing**
1. With this branch checked out verify alerts still work as before on a service landing page and service page (i.e. missing new field doesn't cause issues). Don't use blank titles on your alerts (see #121 )
2. Apply the Drupal changes above.
3. Edit a service page and verify the _Hide alert message_ checkbox appears and is defaulting to unchecked. View the page in question and verify it continues to display the parent alert.
4. Edit the service page again and check the _Hide alert message_ checkbox. Refresh the view of the page and verify that the alert has now disappeared.
